File: code/c/storing-using-data/parameter-test.c
/* Program: parameter-test.c */

void print_equation(int m, double x, int c)
{
    write_line(to_string(m) + " x " + to_string(x) " + " + to_string(c) + " = " + to_string(m * x + c));
}

int main()
{
    print_equation(2, 5.1, 3);
    print_equation(7, 2.74, -8);
    return 0;
}

File: code/pascal/storing-using-data/ParameterTest.pas
program ParameterTest;

procedure PrintEquation(m: Integer; x: Double; c: Integer);
begin
    WriteLn(m, ' x ', x:4.2, ' + ', c, ' = ', m * x + c);
end;

begin
    PrintEquation(2, 5.1, 3);
    PrintEquation(7, 2.74, -8);
end.